server/core/src/https/v1.rs

```diff
@@ -38,7 +37,22 @@ pub(crate) struct SessionId {
     pub sessionid: Uuid,
 }
 
-// /v1/raw/create
+#[utoipa::path(
+    post,
+    path = "/v1/raw/create",
```

Comment:
I like that you added this with openapi, but lets *not* document the /raw commands ... they are there as breakglass for us to issue emergency help commands if needed.

Reply:
Done